### PR TITLE
A few doc updates after discussion on Spectrum

### PR DESCRIPTION
**Pull request by SteveALee**
## Description
Each change is a separate commit and there are few enough to just read the commit messages

## Changes/Tasks
Update to several README.md and added a new one for templates

## Motivation and Context
Improvments to docs

- [ ] Refactoring/add tests (refactoring or adding test which isn't a fix or add a feature)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [ ] My changes have tests around them

### CODE REVIEW

File: docs/concepts.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Deploying a static site has never been easier on today's internet! There are so
 - [Heroku](http://blog.teamtreehouse.com/deploy-static-site-heroku)
 - [AWS (S3 + Cloudfront)](https://aws.amazon.com/getting-started/projects/host-static-website/)
 - [Google Cloud Platform (GCS)](https://cloud.google.com/storage/docs/hosting-static-website)
+- [Zeit now](https://zeit.co/now)
```
Comment (tannerlinsley) [inline docs/concepts.md]: Let's put this at the top of the list.

Comment (tannerlinsley) [reply docs/concepts.md]: Sorry, I didn't see the whole list. Let's put it after Netlify.

**Comment by SteveALee** [comment]
@tanayseven mI moved Zeit not sure if you would have been notified

**Comment by tannerlinsley** [comment]
Yep :) It's merged.

**Pull request merged**